cloud/pkg/controller/controller/downstream.go

```diff
@@ -215,7 +228,15 @@ func NewDownstreamController() (*DownstreamController, error) {
 		return nil, err
 	}
 
-	podManager, err := manager.NewPodManager(cli, v1.NamespaceAll)
+	var nodeName = ""
+	if config.EdgeSiteEnabled {
+		if config.KubeNodeName == "" {
+			return nil, fmt.Errorf("kubeEdge node name is not provided in edgesite controller configuration")
```

Comment:
I think you should use log.LOGGER.Errorf and not fmt.Errorf.

Reply:
This is ok to do fmt.Errorf() because the caller of NewDownstreamController(), is reading this error and logging it. So need to log same information twice.

Reply:
sry I look over the return value of NewDownstreamController